### ISSUE TITLE
Maximum world size?

### ISSUE THREAD

**Issue by ChillerDragon** [fogleman/Craft#264]
Is there a official maximum world size or border? Things get trippy when using commands like ``/pq 99000 99000``

![Screenshot from 2020-05-28 13-10-06](https://user-images.githubusercontent.com/20344300/83134511-c23d3400-a0e4-11ea-814f-22d9389360de.png)
![Screenshot from 2020-05-28 13-10-58](https://user-images.githubusercontent.com/20344300/83134537-cd905f80-a0e4-11ea-9801-9169ea5bba23.png)
![Screenshot from 2020-05-28 13-11-12](https://user-images.githubusercontent.com/20344300/83134545-cf5a2300-a0e4-11ea-9592-a1e045d56a6a.png)
![Screenshot from 2020-05-28 13-11-21](https://user-images.githubusercontent.com/20344300/83134549-d123e680-a0e4-11ea-896f-cd7129cb3495.png)

**Comment by TipzTeam** [comment]
oh wow, maybe just like the real minecraft, you know computers can't count inf numbers.

**Comment by fogleman** [comment]
Yeah, I didn't do anything special to deal with large coordinates, so floating point error becomes a problem far from the origin.

**Comment by ChillerDragon** [comment]
> oh wow, maybe just like the real minecraft, you know computers can't count inf numbers.

Yea sure and minecraft used to have its bugs but today they seem to handle it well.

> Yeah, I didn't do anything special to deal with large coordinates

What about going with the minecraft way and having a world border that makes sure nobody goes too far.

**Comment by TipzTeam** [comment]
just a invisible bedrock wall seems fair, i should code it myself

**Comment by Far117** [comment]
I think just like the farlands, having trippy visuals so far from the origin could be a sort of reward for walking out that far. Imagine how many people would have missed out on something cool if Minecraft walled you in before you could see the farlands.

**Comment by ChillerDragon** [comment]
> I think just like the farlands, having trippy visuals so far from the origin could be a sort of reward for walking out that far. Imagine how many people would have missed out on something cool if Minecraft walled you in before you could see the farlands.

Yes I get your point. But its not such a big reward when you get stuck. In minecraft there is some interesting terrain generation going on but here in Craft you can hardly move. So it can happen that you somehow get that far and then you are trapped :/
